### DISCUSSION TITLE
what are the best recourses for learning data structures

### DISCUSSION THREAD

**Discussion by muhammadShahidk** [community/community#122169] [Programming Help]
### Select Topic Area

Question

### Body

i want to learn data structures. Recently, I have been practicing problem-solving on LeetCode. Can you recommend trusted resources for me to learn from?

**Comment by vinihsr** [comment]
Choose Your Learning Material, Start with Fundamentals, Follow a Structured Approach, Practice Regularly, Understand Algorithms, Analyze Time and Space Complexity, Review and Reinforce, Apply Your Knowledge, Stay Engaged, Seek Help When Needed and Iterate and Improve.

**Comment by dragonwarrior9873** [comment]
Of course learning about the data structure is good way to boost your way of life.
The best materials you should choose to study about the structure is to select relational or non-relational databases and study about that.
That may include mongodb, Mysql, redis, cassandra, etc.
Then you will make successes.
